--- origsrc/lxqt-config-0.8.0/liblxqt-config-cursor/CMakeLists.txt	2014-10-10 14:29:18.000000000 -0500
+++ src/lxqt-config-0.8.0/liblxqt-config-cursor/CMakeLists.txt	2014-11-25 17:15:33.463641100 -0600
@@ -88,5 +88,5 @@ add_library ( lxqt-config-cursor SHARED
 target_link_libraries ( lxqt-config-cursor  ${QTX_LIBRARIES} ${X11_X11_LIB} ${X11_Xcursor_LIB} ${LXQT_LIBRARIES} ${QTXDG_LIBRARIES} ${ZLIB_LIBRARY} ${X11_Xfixes_LIB})
 # not needed probably ${X11_Xfixes_LIB})
 
-install(TARGETS lxqt-config-cursor DESTINATION ${LIB_INSTALL_DIR})
+install(TARGETS lxqt-config-cursor RUNTIME DESTINATION bin LIBRARY DESTINATION ${LIB_INSTALL_DIR})
 install(FILES ${DESKTOP_FILES} DESTINATION "${CMAKE_INSTALL_PREFIX}/share/applications")
